app/javascript/app/components/map/map-component.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import Proptypes from 'prop-types';
+import {
+  ComposableMap,
+  ZoomableGroup,
+  Geographies,
+  Geography
+} from 'react-simple-maps';
+import { Motion, spring } from 'react-motion';
+
+import styles from './map-styles.scss';
+
+class Map extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      center: props.initial.center,
+      zoom: props.initial.zoom
+    };
+  }
+
+  handleZoomIn = () => {
+    this.setState({
+      zoom: this.state.zoom * 2
+    });
+  };
+
+  handleZoomOut = () => {
+    const { zoom } = this.state;
+    if (zoom > 1) {
+      this.setState({
+        zoom: zoom / 2
+      });
+    }
+  };
+
+  render() {
+    const { zoom, center } = this.state;
+    const {
+      paths,
+      zoomEnable,
+      dragEnable,
+      onCountryClick,
+      computedStyles
+    } = this.props;
+    return (
+      <div className={styles.wrapper}>
+        {zoomEnable &&
+          <div className={styles.actions}>
+            <button onClick={this.handleZoomIn}>+</button>
```

Comment:
could you not use the Button component here to include the SVGs from the assets folder for the buttons?